gwt-material/src/test/java/gwt/material/design/client/ui/MaterialModalTest.java

```diff
@@ -41,21 +41,6 @@ public void init() {
         checkType(modal);
         checkDimissible(modal);
         checkDuration(modal);
-        checkMultipleModalZIndexes();
-    }
-
-    protected void checkMultipleModalZIndexes() {
```

Comment:
What you can do is just get the index of each open modal ```$(".lean-overlay").eq(index).css("zIndex")``` this will return the correct zIndex given with the modal index.